src/coda.ts

```diff
@@ -0,0 +1,204 @@
+const formulaNames: Fig.Generator = {
+  script: `grep -A5 --include=\*.ts --exclude-dir=node_modules -r 'addFormula\\|addSyncTable\\|makeFormula\\|makeSyncTable' . | grep -A3 -i formula | grep name: | grep -oE "['\\"]\\w*['\\"]"`,
+  postProcess: (output) => {
+    if (output.trim().length === 0) {
+      return [];
+    }
+    return output.split("\n").map((formulaName) => {
+      return {
+        name: formulaName.replaceAll(/['"]/g, "").trim(),
+        description: `Execute ${formulaName}`,
+      };
+    });
+  },
+};
+
+const commonOptions: Fig.Option[] = [
+  {
+    name: "--version",
+    description: "Show version number",
```

Comment:
This allow the option to be in all the subcommands too without having to add it multiple times
```suggestion
    description: "Show version number",
    isPersistent: true,
```

Reply:
In this way you can avoid having `commonOptions` and we can just add it once to the root command.

Reply:
Thanks for the suggestion. The file is much shorter and cleaner this way! Addressed.